### PR TITLE
shelter: change README and usage according to code logical change abo…

### PR DESCRIPTION
**Pull request by zhiminghufighting**
…ut "--mutual"

Fixes: #1027
Signed-off-by: zhiminghufighting <zhiming.hu@intel.com>

### CODE REVIEW

File: .github/workflows/pr_shelter.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
         mkdir -p /root/inclavare-containers/${{ matrix.tag }};
         cd inclavare-containers-$RUNE_VERSION/enclave-tls;
         make SGX=1 && make install;
-        cp -rf /opt/enclave-tls /root/inclavare-containers/${{ matrix.tag }}'
+        cp -rf /usr/share/enclave-tls /root/inclavare-containers/${{ matrix.tag }}'
```
Comment (haosanzi) [inline .github/workflows/pr_shelter.yml:48]: We install bin into `/usr/share/enclave-tls/samples/`. The libraries and head files are still installed in `/opt/enclave-tls`.  So only copy `/usr/share/enclave-tls` is not enough.

Comment (zhiminghufighting) [reply .github/workflows/pr_shelter.yml:48]: For this directory, i only need enclave-tls-server.bin and stub_enclave_signed.so for test. after make and make install.  Other components are still under /opt/enclave-tls.

Comment (zhiminghufighting) [reply .github/workflows/pr_shelter.yml]: Separate the two commit into two PRs.

File: shelter/README.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ OPTIONS:
    --verifier       set the type of quote verifier(nullquote, sgx_la or sgx_ecdsa)
    --tls            set the type of tls wrapper(nulltls, wolfssl or wolfssl_sgx)
    --crypto         set the type of crypto wrapper(nullcrypto, wolfcrypt or wolfcrypt_sgx)
-   --mutual         set to enable mutual attestation(True, False)
+   --mutual         enable mutual attestation by '--mutual' or disable it by ignoring this option
```
Comment (jiazhang0) [inline shelter/README.md:78]: Just simply "enable mutual attestation" is enough.

Comment (zhiminghufighting) [reply shelter/README.md]: done

**Review by jiazhang0 — APPROVED** [review]

**Pull request merged**